### DISCUSSION TITLE
Skip downloading NPM dependencies if task(s) are already in cache.

### DISCUSSION THREAD

**Discussion by nicksrandall** [vercel/turborepo#2566] [Help] [answered]
When using CI, I'd like to be able to skip downloading NPM dependencies if task is already in cache. Is such a thing possible?

**Comment by nathanhammond** [comment, marked as answer]
`turbo-ignore` is designed to be runnable prior to module installation. In other words, yes, this works, but you'll need to correctly configure your CI runs to be:
- Checkout
- Check ignore
- Install
